CONTRIBUTING.md

```diff
@@ -6,8 +6,8 @@ Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in
 
 ## Pull Request Process
 
-1. Ensure the CI pipeline and all checks are passing.
-2. Update documentation with details of changes.
-3. Add an entry to the [change log](CHANGELOG.md).
+1. Ensure the CI pipeline and all checks are passing
+2. Update documentation with details of changes
+3. Add an entry to the [change log](CHANGELOG.md) using `[Unreleased]` as the version header
```

Comment:
Since this isn't directly related to this PR, can you do this as a different PR?